samples/azure-quantum/resource-estimation/re-with-qir.ipynb

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
Once names of the samples are decided, we may want to align the file names themselves, so from maintenance perspective, it is also easier to know which files map to which samples in this repo (other than looking at mapping file).

I think this one is currently called something similar to re-advanced-analysis,ipynb

Reply:
Makes sense. But I think this comment was meant to be added to what is called now `re-experiments-qsharp.ipynb`.